utils/errors.go

```diff
@@ -32,3 +35,27 @@ func NewForbiddenError(args ...interface{}) error     { return NewError(ErrForbi
 func NewInvalidError(args ...interface{}) error       { return NewError(ErrInvalid, args...) }
 func NewNotFoundError(args ...interface{}) error      { return NewError(ErrNotFound, args...) }
 func NewUnauthorizedError(args ...interface{}) error  { return NewError(ErrUnauthorized, args...) }
+
+type LocError []*i18n.LocalizeConfig
```

Comment:
It would be great to have a godoc comment here explaining what this does, how to use it and why it doesn't implement the `Error` interface